lib/msal-angular/src/msal-guard.service.ts

```diff
@@ -34,13 +34,12 @@ export class MsalGuard implements CanActivate {
                         this.authService.getCacheStorage().setItem(Constants.angularLoginRequest, loginStartPage);
                     }
                     if (this.config.popUp) {
-                        return new Promise((resolve, reject) => {
-                            this.authService.loginPopup(this.config.consentScopes, this.config.extraQueryParameters).then(function (token) {
-                                resolve(true);
+                        return this.authService.loginPopup(this.config.consentScopes, this.config.extraQueryParameters)
```

Comment:
does this solve ie 11?

Reply:
This change is really fix to type errors in typescript...I could fix it the same way `Map` was fixed (by importing a promise polyfill from `core-js-pure` but decided not to since newing up a `Promise` was actually needed from a functional perspective.